server/app/scanpy_engine/scanpy_engine.py

```diff
@@ -115,6 +115,7 @@ def _create_schema(self):
                 "type": str(self.data.X.dtype),
             },
             "annotations": {"obs": [], "var": []},
+            "layout": {"obs": [], "var": []}
```

Comment:
With no plans on the horizon to support var layout, I'd just as soon leave it out.

Reply:
fixed